packages/console/package.json

```diff
@@ -70,7 +71,10 @@
           ]
         }
       }
-    ]
+    ],
+    "rules": {
+      "react/button-has-type": "off"
```

Comment:
wondering why disable this rule? we can have a default `button` type like this:
```
<button type="button">xxx</button>
```

Reply:
this rule only allows explicit string value for button type, e.g. `type: 'button' | 'submit'` won't work. it's more like a js rule. since we're using TypeScript, no need to enable it.